arc-mlir/src/include/arc/Ops.td

```diff
@@ -84,7 +84,7 @@ class ArcFloatUnaryOp<string mnemonic, list<OpTrait> traits = []>
     : ArcUnaryOpSameOperandAndResultType<mnemonic, traits>,
       Arguments<(ins FloatLike:$operand)>;
 
-def SinOp : ArcFloatUnaryOp<"sin"> {
```

Comment:
Why? Upstream uses this convention, so why should we be different?

Reply:
Ok, there were some ops before without the `Op` suffix. To be consistent I can amend so every op has an `Op` suffix. @frej WDYT?

Reply:
Do the amend to make sure we follow the convention.